SupportYourLocals.WPF/MainWindow.xaml.cs

```diff
@@ -177,8 +211,10 @@ private void ButtonSave_Clicked(object sender, RoutedEventArgs e)
 
                 //var dictionaryProducts = convertDictionaryListTextBoxToDictionaryListString(dictionaryOfTextBoxListAddProduct);
                 var dictionaryListString = convertDictionaryListTextBoxToDictionaryListString(dictionaryOfTextBoxListAddProduct);
-                /// This line will be used, when SaveData is changed
-                //CSVData.SaveData(productType, dictionaryListString, MainMap.TargetCenter);
+
+                ///change ID of a local seller
+                ///Change AddedById field
+                xmlData.AddData(new LocationData(1000, MainMap.TargetCenter, AddLocalSellerNameTextBox.Text, 10, DateTime.Now, dictionaryListString));
```

Comment:
1000 for ID and 10 for AddedByID are used only for testing. These fields will be changed when we create new functionality for users' accounts which store ID. And by this ID we will determine AddedByID

Reply:
This is fine for now, but remove testing code before merging.